lib/changelogMarkdownParser.js

```diff
@@ -4,37 +4,29 @@ function parseChangelogMarkdown(markdown) {
     const changelog = {};
     let currentVersion;
     let currentLocale;
-    markdown.split('\n').map(line => line.trim()).forEach((line) => {
-        if (line.search(/^##[^#]+/) !== -1) {
+    markdown.split('\n').forEach((line) => {
+        if (line.search(/^ {0,3}##[^#]+/) !== -1) {
```

Comment:
The changelog format expected by this parser requires that you have no leading space. It might be supported in Markdown, but we don't want to support this since we consider formatting a changelog in this way a mistake.

Also, zero to three spaces seems a bit of an odd choice anyway.

```suggestion
        if (line.search(/^##[^#]+/) !== -1) {
```

Reply:
> Also, zero to three spaces seems a bit of an odd choice anyway.

No, when using 4 spaces, the line is not a header anymore but becomes a code block.